File: Resources/TOI-Zero_68/A1/013/013.py
c = input().strip()
s = input().strip()

if c == 'H' and s == "4567":
    print("safe unlocked")
elif s == "4567":
    print("safe locked - change char")
elif c == 'H':
    print("safe locked - change digit")
else:
    print("safe locked")

File: Resources/TOI-Zero_68/A1/013/013.c
#include <stdio.h>
#include <string.h>

int main() {
    char c;
    char s[100];
    scanf(" %c", &c);
    scanf("%s", s);
    
    if (c == 'H' && strcmp(s, "4567") == 0) {
        printf("safe unlocked\n");
    } else if (strcmp(s, "4567") == 0) {
        printf("safe locked - change char\n");
    } else if (c == 'H') {
        printf("safe locked - change digit\n");
    } else {
        printf("safe locked\n");
    }
    
    return 0;
} 